recipes/DABEST/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dabest" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8147e304ebc587923f0e2692ff68eb33e5abba81d0b93994da1d14cc4479dec4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - matplotlib
+    - seaborn
+    - pandas
+    - numpy
+    - scipy
+    - pytest
+
+test:
+  imports:
```

Comment:
You should only test the import of the package in the recipe, not its dependencies. Currently you're not doing the latter and only the former. It should look like:

```
tests:
   imports:
      - dabest
```